### PR TITLE
Improved organization of generated code

### LINKED ISSUE #214: Move all custom rust code into the generators folder

### PR DESCRIPTION
**Pull request by dansteren**
Moved all generated code (i.e. `quote`s) inside of the [generators](https://github.com/demergent-labs/kybra/tree/main/kybra/compiler/kybra_generate/src/generators) folder.

Closes #214

### CODE REVIEW

File: kybra/compiler/kybra_generate/src/generators/kybra_serde.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@
```
Comment (lastmjs) [inline kybra/compiler/kybra_generate/src/generators/kybra_serde.rs:24]: Delete this whole thing

File: kybra/compiler/kybra_generate/src/generators/header.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+pub fn generate() -> proc_macro2::TokenStream {
```
Comment (lastmjs) [inline kybra/compiler/kybra_generate/src/generators/header.rs:1]: Please make this look like Azle with a directory and such

File: kybra/compiler/kybra_generate/src/generators/func.rs [hunk 3]
```diff
@@ -0,0 +1,49 @@
+use proc_macro2::TokenStream;
```
Comment (lastmjs) [inline kybra/compiler/kybra_generate/src/generators/func.rs:1]: Please change try_from_vm_value to try_from_vm_value_impls

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: kybra/compiler/kybra_generate/src/generators/func.rs (hunk 3 above)
Comment (dansteren) [reply kybra/compiler/kybra_generate/src/generators/func.rs:1]: Done with 1f4dcde16881249266f41c1f6cb87d3701a4181f

File: kybra/compiler/kybra_generate/src/generators/kybra_serde.rs (hunk 1 above)
Comment (dansteren) [reply kybra/compiler/kybra_generate/src/generators/kybra_serde.rs:24]: Done in 55d1d1dba73234b8b83a0c26c7166f437fe6f7de

File: kybra/compiler/kybra_generate/src/generators/header.rs (hunk 2 above)
Comment (dansteren) [reply kybra/compiler/kybra_generate/src/generators/header.rs:1]: Done in e2647202f3a44245d6719d104f7cb79c4f85e783

File: kybra/compiler/kybra_generate/src/generators/canister_methods/init.rs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                    let err_string: String = err.to_pyobject(vm).repr(vm).unwrap().to_string();
+
+                    panic!("{}", err_string);
+                }
+            });
+
+            _KYBRA_INTERPRETER_OPTION = Some(_kybra_interpreter);
+            _KYBRA_SCOPE_OPTION = Some(_kybra_scope);
+
+            #ic_object
```
Comment (lastmjs) [inline kybra/compiler/kybra_generate/src/generators/canister_methods/init.rs:46]: Don't repeat this

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: kybra/compiler/kybra_generate/src/generators/canister_methods/init.rs (hunk 4 above)
Comment (dansteren) [reply kybra/compiler/kybra_generate/src/generators/canister_methods/init.rs]: Resolved in 9b2c71d3332f699c06d76db8977902ec74f80540

**Review by lastmjs — APPROVED** [review]

**Pull request merged**